app/src/main/java/com/example/warehousemanagement/domain/model/ImportPackages.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.warehousemanagement.domain.model
+
+import java.util.Date
+
+data class ImportPackages(
+    val id: String,                // Định danh gói nhập
+    val packageName: String,        // Tên gói nhập
+    val importDate: Date,           // Ngày nhập hàng
+    val supplier: Supplier,         // Thông tin nhà cung cấp
+    val idReceivedBy: String,       // Người nhận hàng
+    val status: Status= Status.PENDING,  // Tình trạng gói nhập, default is PENDING
+    val note: String? = null        // Ghi chú, optional
```

Comment:
What does this package contain? . Which product and quatity.

Reply:
Done

Reply:
Done what :( , add ` val listProduct: List<Product>`

Reply:
do the same thing with `export`